source/contracts/reporting/ReputationToken.sol

```diff
@@ -53,14 +53,9 @@ contract ReputationToken is DelegationTarget, ITyped, Initializable, VariableSup
     function migrateIn(address _reporter, uint256 _attotokens) public afterInitialized returns (bool) {
         IUniverse _parentUniverse = universe.getParentUniverse();
         require(ReputationToken(msg.sender) == _parentUniverse.getReputationToken());
+        require(controller.getTimestamp() < _parentUniverse.getForkEndTime());
```

Comment:
Note, not feedback: Instead of checking that current time is before forkEnd time to provide bonus, simply require it. All rep not moved before this time will cause a tx revert, locking REP in parent universe.